### ISSUE TITLE
Beta 2: New Docker deployment has Embedded MariaDB issue

### ISSUE THREAD

**Issue by mickeykarimzadeh** [louislam/uptime-kuma#5389]
### 📑 I have found these related issues/pull requests

https://github.com/louislam/uptime-kuma/issues/5279
https://github.com/louislam/uptime-kuma/issues/5353

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

Setting up a new instance, embedded MariaDB gives errors when trying to setup

### 👟 Reproduction steps

New AWS EC2 instance, t4g.nano, Ubuntu Server 24.04 (ARM64)

sudo apt update
sudo apt install docker.io -y
sudo docker run -d --restart=always -p 3001:3001 -v uptime-kuma:/app/data --name uptime-kuma louislam/uptime-kuma:nightly2

Open browser and go to http://x.x.x.x:3001
Select embedded MariaDB

### 👀 Expected behavior

New instance of Uptime Kuma ready to use

### 😓 Actual Behavior

Showing the Docker logs with
sudo docker logs -f uptime-kuma

shows a looping error of MariaDB failing to start with an error

### 🐻 Uptime-Kuma Version

2.0.0-beta.0-nightly-20241105100618

### 💻 Operating System and Arch

Ubuntu Server 24.04 (ARM64)

### 🌐 Browser

Google Chrome

### 🖥️ Deployment Environment

N/A

### 📝 Relevant log output

```shell
Welcome to Uptime Kuma
Your Node.js version: 20.18.0
2024-11-28T00:59:26Z [SERVER] INFO: Env: production
2024-11-28T00:59:28Z [SERVER] INFO: Uptime Kuma Version: 2.0.0-beta.0-nightly-20241105100618
2024-11-28T00:59:28Z [SERVER] INFO: Loading modules
2024-11-28T00:59:30Z [SERVER] INFO: Creating express and socket.io instance
2024-11-28T00:59:30Z [SERVER] INFO: Server Type: HTTP
2024-11-28T00:59:31Z [SERVER] INFO: Data Dir: ./data/
2024-11-28T00:59:31Z [SETUP-DATABASE] INFO: db-config.json is not found or invalid: ENOENT: no such file or directory, open 'data/db-config.json'
2024-11-28T00:59:31Z [SETUP-DATABASE] INFO: Starting Setup Database on 3001
2024-11-28T00:59:31Z [SETUP-DATABASE] INFO: Open http://localhost:3001 in your browser
2024-11-28T00:59:31Z [SETUP-DATABASE] INFO: Waiting for user action...
Request /setup-database-info
2024-11-28T00:59:50Z [SETUP-DATABASE] INFO: Database is configured, close the setup-database server and start the main server now.
2024-11-28T00:59:50Z [SETUP-DATABASE] INFO: The setup-database server is closed
2024-11-28T00:59:50Z [DB] INFO: Database Type: embedded-mariadb
2024-11-28T00:59:50Z [MARIADB] INFO: Embedded MariaDB: /app/data/mariadb is not found, create one now.
2024-11-28T01:00:36Z [MARIADB] ERROR: 2024-11-28  0:59:50 0 [Warning] mariadbd: io_uring_queue_init() failed with errno 95
2024-11-28  0:59:50 0 [Warning] InnoDB: liburing disabled: falling back to innodb_use_native_aio=OFF
2024-11-28  0:59:51 0 [Warning] You need to use --log-bin to make --expire-logs-days or --binlog-expire-logs-seconds work.
Killed

2024-11-28T01:00:36Z [MARIADB] INFO: Starting Embedded MariaDB
2024-11-28T01:00:36Z [MARIADB] INFO: 2024-11-28  1:00:36 0 [Note] Starting MariaDB 10.11.6-MariaDB-0+deb12u1 source revision  as process 72

2024-11-28T01:00:37Z [MARIADB] INFO: 2024-11-28  1:00:36 0 [Note] mariadbd: Aria engine: starting recovery
recovered pages: 0% 10% 21% 33% 43% 54% 67% 80% 90% 100% (0.0 seconds); tables to flush: 5 4 3 2 1 0
 (0.0 seconds);
2024-11-28  1:00:36 0 [Note] mariadbd: Aria engine: recovery done
2024-11-28  1:00:36 0 [Note] InnoDB: Compressed tables use zlib 1.2.13
2024-11-28  1:00:36 0 [Note] InnoDB: Number of transaction pools: 1
2024-11-28  1:00:36 0 [Note] InnoDB: Using ARMv8 crc32 + pmull instructions
2024-11-28  1:00:36 0 [Note] mariadbd: O_TMPFILE is not supported on /tmp (disabling future attempts)
2024-11-28  1:00:36 0 [Warning] mariadbd: io_uring_queue_init() failed with errno 95
2024-11-28  1:00:36 0 [Warning] InnoDB: liburing disabled: falling back to innodb_use_native_aio=OFF
2024-11-28  1:00:36 0 [Note] InnoDB: Initializing buffer pool, total size = 128.000MiB, chunk size = 2.000MiB
2024-11-28  1:00:36 0 [Note] InnoDB: Completed initialization of buffer pool
2024-11-28  1:00:36 0 [Note] InnoDB: File system buffers for log disabled (block size=512 bytes)
2024-11-28  1:00:36 0 [Note] InnoDB: Starting crash recovery from checkpoint LSN=17071
2024-11-28  1:00:36 0 [Note] InnoDB: End of log at LSN=46680
2024-11-28  1:00:36 0 [Note] InnoDB: To recover: 21 pages
2024-11-28  1:00:36 0 [Note] InnoDB: 128 rollback segments are active.
2024-11-28  1:00:36 0 [Note] InnoDB: Removed temporary tablespace data file: "./ibtmp1"
2024-11-28  1:00:36 0 [Note] InnoDB: Setting file './ibtmp1' size to 12.000MiB. Physically writing the file full; Please wait ...
2024-11-28  1:00:36 0 [Note] InnoDB: File './ibtmp1' size is now 12.000MiB.
2024-11-28  1:00:36 0 [Note] InnoDB: log sequence number 46680; transaction id 14
2024-11-28  1:00:36 0 [Note] InnoDB: Loading buffer pool(s) from /app/data/mariadb/ib_buffer_pool
2024-11-28  1:00:36 0 [Note] InnoDB: Cannot open '/app/data/mariadb/ib_buffer_pool' for reading: No such file or directory
2024-11-28  1:00:36 0 [Note] Plugin 'FEEDBACK' is disabled.

2024-11-28T01:00:37Z [MARIADB] INFO: 2024-11-28  1:00:37 0 [Warning] You need to use --log-bin to make --expire-logs-days or --binlog-expire-logs-seconds work.

2024-11-28T01:00:37Z [MARIADB] INFO: Waiting for Embedded MariaDB to start...
2024-11-28T01:00:37Z [MARIADB] INFO: 2024-11-28  1:00:37 0 [Note] Server socket created on IP: '127.0.0.1'.
2024-11-28  1:00:37 0 [ERROR] Can't start server : Bind on unix socket: No such file or directory
2024-11-28  1:00:37 0 [ERROR] Do you already have another server running on socket: /app/data/run/mariadb.sock ?
2024-11-28  1:00:37 0 [ERROR] Aborting

2024-11-28T01:00:38Z [MARIADB] INFO: Stopped Embedded MariaDB: 1
2024-11-28T01:00:38Z [MARIADB] ERROR: Try to restart Embedded MariaDB as it is not stopped by user
2024-11-28T01:00:38Z [MARIADB] INFO: Starting Embedded MariaDB
```

**Comment by mickeykarimzadeh** [comment]
I did the same commands on Debian 12 ARM and it worked.

**Comment by louislam** [comment]
As you have found, it is duplicate of https://github.com/louislam/uptime-kuma/issues/5279.

Will be fixed in the next beta, please follow #5279 for the latest progress.
